Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "MIT"
 
 [dependencies]
 hyper = { version = "0.9", default-features = false }
-regex = "0.1"
+regex = "0.1.59"
```

Comment:
Is there a specific reason you used this version? I was able to compile with 0.1.80 just fine.